MIDAS/src/hardware/Voltage.cpp

```diff
@@ -1,17 +1,19 @@
 #include "sensors.h"
+#include <Arduino.h>
 
-// #include sensor library
-
-// global static instance of the sensor
-
-
+/**
+ * "Initializes" the voltage sensor. Since it reads directly from a pin without a library, there is no specific initialization.
+*/
 ErrorCode VoltageSensor::init() {
-    // do whatever steps to initialize the sensor
-    // if it errors, return the relevant error code
     return ErrorCode::NoError;
 }
 
+/**
+ * Reads the value of the given analog pin and converts it to a battery voltage with the assumption that the voltage sensor is plugged into that pin
+ * \return The scaled voltage given by the voltage sensor
+*/
 Voltage VoltageSensor::read() {
-    // read from aforementioned global instance of sensor
-    return Voltage();
+    Voltage v_battery;
+    v_battery.voltage = analogRead(69) / 1024.f * 3.3f * 3.f;
```

Comment:
Can you make the 69 into a #define so it's easier to find when we get the real pins